catalog-rest-service/src/main/resources/json/data/tests/columnValueMaxToBeBetween.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "columnValueMaxToBeBetween",
+    "displayName": "columnValueMaxToBeBetween",
+    "description": "This schema defines the test ColumnValueMaxToBeBetween. Test the maximum value in a col is within a range.",
+    "testPlatforms": ["OpenMetadata", "GreatExpectations"],
```

Comment:
@harshach should we keep `GreatExpectations` here? I am thinking, once we have this new generic test suite to update the `great_expectations` submodule to not pick up those test cases anymore but to create generic tests on the fly. wdyt?

Reply:
@TeddyCr I think it would be good to have platform to show in the UI where the tests were running. I can move it from testDefinition to testCase  wdyt?

Reply:
Good point. Makes total sense.